tests/serial/time/implicit/serial-coupling/WaveformSubsyclingWithConstantAcceleration.cpp

```diff
@@ -0,0 +1,135 @@
+#ifndef PRECICE_NO_MPI
+
+#include <math.h>
+#include <precice/precice.hpp>
+#include <vector>
+#include "testing/Testing.hpp"
+
+using namespace precice;
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(Time)
+BOOST_AUTO_TEST_SUITE(Implicit)
+BOOST_AUTO_TEST_SUITE(SerialCoupling)
+
+/**
+ * @brief Simple test to ensure that underrelaxation is applied to every  timestep.
```

Comment:
```suggestion
 * @brief Simple test to ensure that underrelaxation is applied to every time step.
```
Please write `time step` (in text) and `timeStep` (in code). We agreed on this at some point in time to make things consistent. See https://github.com/precice/precice/pull/1628 for some more spelling rules.

Similar for `time window` (in text) and `timeWindow` (in code).